app/views/govuk_publishing_components/components/_previous_and_next_navigation.html.erb

```diff
@@ -2,12 +2,16 @@
   add_gem_component_stylesheet("previous-and-next-navigation")
 
   if local_assigns.include?(:next_page) || local_assigns.include?(:previous_page)
+    ga4_tracking ||= false
```

Comment:
I'd move this above line 4 as I don't think it should be part of the `if` statement.